arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,536 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "description": "Gets result of the specified operation",
+        "tags": [ "RpProxy" ],
+        "operationId": "RpProxy_GetOperationResultAsync",
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Identifier for the operation for which to get status"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          },
+          "204": {
+            "description": "No Content",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RpServerUpdateRequest": {
+      "description": "Update request specification",
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location of the service"
+        },
+        "sku": {
+          "$ref": "#/definitions/RpSkuTier",
+          "description": "Server SKU"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "Key value pairs"
+          },
+          "description": "tags associated with analysis server"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RpPropertyBag",
+          "description": "Property bag for resource operation"
+        }
+      }
+    },
+    "RpSkuTier": {
+      "description": "Represents the SKU name and tier",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Sku name for frontend requests"
+        },
+        "tier": {
+          "type": "string",
+          "description": "Tier for frontend requests"
+        }
+      }
+    },
+    "RpPropertyBag": {
+      "description": "Property bag",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
```

Comment:
provisioningState should be marked as readonly.
You are referencing RpPropertyBag from RpServerUpdateRequest which is the body parameter for PATCH operation.
So, does that mean
- provisioningState is updatable?

"as Administrators" has a property named members which is an array of strings. So the only way one can update the array would be by adding or removing elements from the array but not modifying/updating the strings in the Array. Is my understanding correct?